### ISSUE TITLE
Увеличение эффективности доставки

### ISSUE THREAD

**Issue by Onixjfc** [willmann/dev360#33]
Надо что бы в плане доставки на каждый день серым цветов выводили все счета которые можно забрать и все заказы которые можно доставить исходя из данных, это будет лишняя напоминалка и проверка для логистов чтобы не проебывать ничего и способ контроля для меня - сразу видно что что-то не доставляется или не забирается зачем то. Причем заказы и с счета выводятся во все дни а нетолько в один (т.е. если заказ не поставили в фактический план 19го, а он мог быть 19го доставлен, то при просмотре 20, 21,23го чисел во всех них серым будет гореть этот заказ, т.к. его можно доставить, пока не будет поставлена огрузка, тоже самое со счетами.

**Comment by willmann** [comment]
Давай договоримся, что мы вначале задачи обсуждаем, а затем заносим сюда. Это первое.
Второе — это приведет к тому, что План доставки будет нечитаем. Я против.

**Comment by Onixjfc** [comment]
Мы договорились что я переношу сюда все из айти-задач, давай так и делать, обсуждать туту как раз удобно - не надо договриваться на часовые созвоны, а можно постепенно.  Давай обсуждать. План читаем будет, достаточно будет небольшого окошка где нить внизу где 2 списка: 1 - номера заказов (ссылки на заказы), 2 список - номера счетов на забор (тоже ссылки). Но это даст огромный эффект, логист сделал план и видит что остались какие то заказы которые горят и манагерам это напоминание (это спасет от того что логисты халявят - ведь менеджеры не знают что что то можно забрать и я этого сразу не вижу, надо изучать, а тут все наглядно) - это еще один шаг к автоматизации и исключения ошибок

**Comment by willmann** [comment]
Но ты представляешь какой огромный объем счетов поставщика у нас не попадают в план? Для этого достаточно зайти в План забора товара. Все эти счета будут внизу. Это даст допинфо, но понизит фокус на основной части Плана доставок — Плане доставок.

**Comment by Onixjfc** [comment]
Ну не скажи, почему ты решил что много ? Если их много это значит что кто-то работает плохо. Выводится должны те счет которые можно начиная с этого дня. Вообще то у нас правило есть по которому действует логистика: мы доставляем все возможные заказы в день когда это возможно и забираем все возможные счета в день когда это возможно, ели это не так - это точно приведет к срывам доставки (3-5 дней) - так что это хорошо, если это будет видно наглядно

**Comment by willmann** [comment]
Ну в смысле ты думаешь их вывести под нашей плиткой с водителями и их точками? По сути объединить два инструмента?

**Comment by Onixjfc** [comment]
Не объединить. Суть в том что там выводятся только те заказы и счета, которые можно было доставить но их не добавили в план, проверка и показатель не халявят ли логсты в каждый конкретный день и насколько эффективно они забирают. Прикольно было бы вывести езе циферку посередине наверху - эффективность в конкретный день (процент сделанного к возможному) -  по сути это некий чек. А в инструментах "план забора" - это инструмент дальнего планирования. Плюс есть еще один очень важный плюс этой штуки ! - Логисты часто планируют забор каких то счетов ставят даты вперед а потом оказывается что их забрать можно раньше, дату готовности меняют на меньшею, а забора нет, а тут будет напоминалка когда загорится серым в какой то день

**Comment by willmann** [comment]
ОК, сделай в Фотошопе пример, как ты это видишь?

**Comment by Onixjfc** [comment]
![default](https://cloud.githubusercontent.com/assets/11632756/6842635/80cece68-d3ab-11e4-9cd2-d0e01e7db925.jpg)

**Comment by willmann** [comment]
ОК, но кликабельными мы сделаем только заказы и максимум номера счетов. Добавлять отсюда наименования в расписание нельзя будет. Норм?

**Comment by Onixjfc** [comment]
да, ок, просто так заходишь  и сразу видно какие вопросы задавать, почему это не доставили? почему про это забыли ? и т.д. и т.п. НО ! Из этого списка должны убираться те что добавили в план. И самый главный момент! Надеюсь ты меня услышал - этот инструмент для того чтобы понять не только про что забыли но и те что доставляют не вовремя, т.е. какой то счет стоит на забор через месяц а готовность на сегодня - он должен быть в этом списке как напоминалка... И заказы тоже, стоит что заказ отвозим через 2 дня, а его можно сегодня на самом деле - он должен быть в этом списке. Понимаешь ????

**Comment by Onixjfc** [comment]
Т.е. видно что логисты что-то что можно забрать и доставить переносят зачем то

**Comment by willmann** [comment]
Давай подытожу — 2 колонки:
1) заказы, по которым все счета забраны и они могут быть доставлены сегодня, но для них:
— не проставлена отгрузка
— отгрузка проставлена на более позднюю дату
2) счета, у которых дата готовности сегодня или ранее, но:
— дата забора не проставлена
— дата забора проставлена на поздние даты

**Comment by Onixjfc** [comment]
четко, только ты это сделай юзерабильно как ты умеешь ))) неплохо было бы показать на какие даты поставлено тоже. Но главное это вес и обьем. Кстати такая задача стотит по всему плану доставок, логиста надо вес и обьем по заказам и счетам обязательно и суммы чтобы понмиать поместиться в тачки или нет
